### DISCUSSION TITLE
How to install code-server without sudo?

### DISCUSSION THREAD

**Discussion by PotatoThanh** [coder/code-server#2039] [Q&A] [answered]
Hi,

I want to install code-server in a remote computer but I do not have sudo privilege. How can I install it? Thanks!

**Comment by Bongani** [comment, marked as answer]
What about the yarn/npm option, that does not require sudo

https://github.com/cdr/code-server/blob/v3.5.0/doc/install.md#yarn-npm

**Reply by PotatoThanh** [reply]
It worked. Thanks!
